.github/workflows/deploy.yml

```
@@ -99,6 +104,7 @@ jobs:
       run: |
         chmod +x ../../tools/scripts/Release.sh
         ../../tools/scripts/Release.sh > ./dists/stable/Release
+        cat ./dists/stable/Release | gpg --default-key neo-project -abs --clearsign --passphrase ${{ secrets.GPG_KEY_PASSWORD }} --pinentry-mode loopback --batch > ./dists/stable/InRelease
```

```suggestion
        cat ./dists/stable/Release | gpg --batch --yes \
    --pinentry-mode loopback \
    --default-key \
    --passphrase "${{ secrets.GPG_KEY_PASSWORD }}" \
    --local-user neo-project \
    --clearsign \
    -o ./dists/stable/InRelease \
    ./dists/stable/Release
```
`--passphrase should be at the beginning, otherwise password will not be read before `--clearsign` execution